File: 프로그래머스/lv3/42884. 단속카메라/단속카메라.py
def solution(routes):
    cnt = 0
    routes.sort(key = lambda x : x[1])
    cam = -30001
    for i in range(len(routes)):
        if routes[i][0] <= cam and cam <= routes[i][1]:
            continue
        else:
            cam = routes[i][1]
            cnt+=1
    return cnt

File: 프로그래머스/3/42884. 단속카메라/단속카메라.js
function solution(routes) {
    routes.sort((a,b)=>a[1]-b[1])
    let cnt=0
    let cam=-30000
    for(let route of routes){
        if(cam>=route[0] && cam<=route[1]) continue
        else {
            cam=route[1]
            cnt++
        }
    }
    return cnt
}